src/python/docs/installation.rst

```diff
@@ -0,0 +1,51 @@
+.. highlight:: shell
+
+============
+Installation
+============
+
+
+Stable release
+--------------
+
+To install DeepOps CLI, run this command in your terminal:
+
+.. code-block:: console
+
+    $ pip install deepops
```

Comment:
It seems that, unfortunately, the deepops PyPi package is already taken https://pypi.org/project/deepops